index.bs

```diff
@@ -1798,6 +1822,66 @@ Note: Invoking this method from a [=browsing context=] where the [=Web Authentic
 
 </div>
 
+### Client Supported Capabilities (dictionary <dfn dictionary>SupportedCapabilities</dfn>) ### {#dictionary-supportedcapabilities}
+
+<xmp class="idl">
+    dictionary SupportedCapabilities {
+        boolean areU2FDevicesSupported;
+        boolean areCTAPDevicesSupported;
+        boolean areResidentKeysSupported;
+        boolean isUserVerificationSupported;
+        boolean isTokenBindingSupported;
+        UserVerifyingPlatformAuthenticatorAvailability userVerifyingPlatformAuthenticatorAvailability;
+        sequence<AuthenticatorTransport> transportsSupported;
+        sequence<DOMString> extensionsSupported;
+    };
+</xmp>
+
+This dictionary is used by [=[WRPS]=] to determine the supported capabilities of the [=client=].
+
+<div dfn-type="dict-member" dfn-for="SupportedCapabilities">
+    :   <dfn>areU2FDevicesSupported</dfn>
+    ::  This attribute indicates whether the [=client=] supports communicating with [[FIDO-U2F-Message-Formats]] devices.
+
+    :   <dfn>areCTAPDevicesSupported</dfn>
+    ::  This attribute indicates whether the [=client=] supports communicating with [[FIDO-CTAP]] devices.
+
+    :   <dfn>areResidentKeysSupported</dfn>
+    ::  This attribute indicates whether the [=client=] supports creating [=resident credentials=].
+
+    :   <dfn>isUserVerificationSupported</dfn>
+    ::  This attribute indicates whether the [=client=] supports [=user verification=].
+
+    :   <dfn>isTokenBindingSupported</dfn>
+    ::  This attribute indicates whether the [=client=] supports the [=Token Binding=] protocol.
+
+    :   <dfn>userVerifyingPlatformAuthenticatorAvailability</dfn>
+    ::  This attribute indicates the availability of a [=user-verifying platform authenticator=] on the [=client=].
+
+    :   <dfn>transportsSupported</dfn>
+    ::  This attribute indicates the {{AuthenticatorTransport}} values supported by the [=client=].
+
+    :   <dfn>extensionsSupported</dfn>
+    ::  This attribute indicates the [=extension identifiers=] supported by the [=client=].
+</div>
+
+### Capability Support - PublicKeyCredential's `getSupportedCapabilities()` Method ### {#sctn-getSupportedCapabilities}
+
+<div link-for-hint="WebAuthentication/getSupportedCapabilities">
+
+[=[WRPS]=] use this method to determine what capabilities are supported by the [=client=].
+Upon invocation, the [=client=] employs a [=client platform=]-specific procedure to discover supported capabilities.
+The promise is resolved with the capabilities that are supported.
+Based on the result, the [=[RP]=] can take further actions to guide the user to create a credential.
```

Comment:
RP may also invoke this method for getting a credential.

Reply:
Agreed, this is also useful for login scenarios.  I can update the text to reflect that.